components/renku_data_services/solr/solr_client.py

```diff
@@ -70,6 +66,13 @@ def __str__(self) -> str:
         )
 
 
+class SolrClientException(Exception, ABC):
```

Comment:
suggestion. We have a "central" parent error for all errors in the data services. I think it would be good to just inherit from that here. It is in `errors.errors`. That error is very similar and it has a message field. It is also nicely handled in our centralized error handling.